test/docs/errors/resourceObjectFields/invalidApiDocumentRelationshipsIdNotRequired.js

```diff
@@ -0,0 +1,1721 @@
+/* eslint-env mocha */
+/* eslint-disable quotes */
+const invalidApiDocumentRelationshipsIdNotRequired ={
```

Comment:
This line is generating linting errors. It can be resolved by running the following command:

`eslint **/*.js /test`